website/content/docs/concepts/auditing.mdx

```diff
@@ -101,19 +101,40 @@ Any session recording metadata that is attached to the storage bucket is deleted
 The BSR (Boundary Session Recording) defines a hierarchical directory structure of files and a binary file format.
 It contains all the data transmitted between a user and a target during a single session.
 
-Boundary creates the top-level directory of the BSR as `<sessionID>.bsr`. This top level directory contains session summary information and subdirectories for connections.
+Boundary creates the top-level directory of the BSR as `<sessionRecordingID>.bsr`. This top level directory contains session summary
+information and subdirectories for connections.
 
 A BSR connections directory contains a summary of connections, as well as inbound and outbound requests.
 If you use a multiplexed protocol, there are subdirectories for the channels.
 
+Every directory contains a SHA256SUMS and SHA256SUMS.sig file, to be used for cryptographically verifying the contents of
+that directory. The SHA256SUMS file contains rows of file names paired with a checksum for file contents. The
```

Comment:
It might be worth noting that the SHA256SUMS file is compatible with [sha256sum](https://linux.die.net/man/1/sha256sum) and that the `sha256sum` command line tool could be used to check files against the checksums? Or maybe that is something that should be added to the `validate-session-recordings` doc?

Reply:
Thanks, I can add that to `validate-session-recordings`!

Reply:
Added!